### PR TITLE
Adds Paramedic Room, Blueshield and Cryodorms to Omega Station

### PR DESCRIPTION
**Pull request by AndrewMontagne**
**Paramedic Room (South of Medbay):**

image

**Cryodorms (North Section of Locker Room):**

image

**Blueshield (Spawns on Bridge):**

image

:cl: AndrewMontagne
add: Added Cryodorms, Paramedic Room and Blueshield Office to Omega Station.
add: Added crew monitoring desktop.
fix: Fixed layering issue with desktop computers.
fix: Waste loop air injector now works.
/:cl:

### CODE REVIEW

**Comment by SamHPurp** [comment]
You need to map merge! 😱 

Also could you please include a spawn point for the Brig Doc, which I believe wasn't mapped in yet.

**Comment by AndrewMontagne** [comment]
[Map rooms for Blueshield, Paramedic, Cryostorage. Move Detective's office?](https://trello.com/c/QPJYO7Ks/53-map-rooms-for-blueshield-paramedic-cryostorage-move-detectives-office)

File: code/modules/stock_market/computer.dm [hunk 1]
```diff
@@ -9,6 +9,7 @@
 	circuit = /obj/item/circuitboard/computer/stockexchange
 	clockwork = TRUE //it'd look weird
 	pixel_y = 8
+	layer = 3.3
```
Comment (SamHPurp) [inline code/modules/stock_market/computer.dm]: Do we not have defines for these?

File: code/modules/library/lib_machines.dm [hunk 2]
```diff
@@ -21,6 +21,7 @@
 	icon_screen = "generic"
 	icon_keyboard = "desktop_key"
 	pixel_y = 8
+	layer = 3.3
```
Comment (SamHPurp) [inline code/modules/library/lib_machines.dm]: Do we not have defines for these?

File: code/game/machinery/computer/security.dm [hunk 3]
```diff
@@ -43,6 +43,7 @@
 	icon_screen = "security"
 	icon_keyboard = "desktop_key"
 	pixel_y = 8
+	layer = 3.3
```
Comment (SamHPurp) [inline code/game/machinery/computer/security.dm]: Do we not have defines for these?

File: code/game/machinery/computer/medical.dm [hunk 4]
```diff
@@ -619,3 +619,4 @@
 	icon_screen = "medcomp"
 	icon_keyboard = "desktop_key"
 	pixel_y = 8
+	layer = 3.3
```
Comment (SamHPurp) [inline code/game/machinery/computer/medical.dm]: Do we not have defines for these?

File: code/game/machinery/computer/crew.dm [hunk 5]
```diff
@@ -13,6 +13,15 @@
...
 	icon_keyboard = "syndie_key"
 
+/obj/machinery/computer/crew/desktop
+	name = "crew monitoring desktop"
+	icon = 'icons/obj/modular_desktop.dmi'
+	icon_state = "desktop-off"
+	icon_screen = "operating"
+	icon_keyboard = "desktop_key"
+	pixel_y = 8
+	layer = 3.3
```
Comment (SamHPurp) [inline code/game/machinery/computer/crew.dm]: Do we not have defines for these?

**Comment by SamHPurp** [comment]
- [ ] Cryodorms

- [ ] Blueshield

- Sorry I really don't like the location of the Blueshield office. I'm also still fairly against bluey having his own office, but this eats into the symmetry of the bridge for what seems to be a really insignificant gain. Perhaps we can just map his locker in somewhere for the time being instead...?

- [ ] Paramedic

- Issue:
image

- Multiple wires down on a single tile

- Multiple of the same pipe down on a single tile

- Supply and scrubber pipes down on the same tile (Paradise can do this, TG cannot... these need to be mapped on separate tiles). (Did it not give you a warning in DreamDaemon when you booted this up to test it?)

**Comment by SamHPurp** [comment]
you've left the original map file in the PR :P

**Comment by AndrewMontagne** [comment]
Fixed, @SamHPurp :P

**Pull request merged**